internal/testgen/gen_negative_positive.go

```diff
@@ -0,0 +1,232 @@
+package main
+
+import (
+	"strings"
+	"text/template"
+
+	"github.com/Antonboom/testifylint/internal/checkers"
+)
+
+type NegativePositiveTestsGenerator struct{}
```

Comment:
Fix typo

Reply:
Thanks, I feel ashamed, haha

Reply:
BTW @Antonboom , an idea for another issue/PR, would you like me to help you in setting up code spell GitHub action for checking typos in the code and documentation ?

Reply:
I am using GoLand that highlights any issues.
Also `misspell` linter is already enabled, maybe need to configure it properly 🤷‍♂️ 

But I dont want to introduce new tools and tries to keep the project as simple as possible (but yep, this is already monstrous application).